packages/react-component-library/src/components/Radio/Radio.stories.tsx

```diff
@@ -23,6 +23,7 @@ const MultipleItemsTemplate: ComponentStory<typeof Radio> = (props) => {
   function getProps(i: number): RadioProps {
     return {
       ...props,
+      variant: CHECKBOX_RADIO_VARIANT.NO_CONTAINER,
```

Comment:
This was breaking when opening the canvas in a new window (because Storybook was deciding to omit the `variant` arg for some reason), so moved this here to work around that.